### ISSUE TITLE
Links not highlighted when they have a class

### ISSUE THREAD

**Issue by gosko** [w3c/w3c-website#747]
When tabbing through links, the focus highlighting is not shown if the links have a class, due to this rule in the CSS:

```css
a:not([class]):focus, .heading-anchor:focus,
a.with-icon--before:focus,
a.with-icon--after:focus {
  background-color: #f9dc4a;
  background-image: linear-gradient(to top, #000 3px, #f9dc4a 3px, #f9dc4a);
  color: #000;
  text-decoration: none;
}
```

I'm using class names on links on an internal application (see the 'Sort by:' links on [this example page](https://www.w3.org/2025/07/moderation-example.html)) and they are missing the highlighting when in focus.

As a temporary workaround I added this to the local CSS:

```css
a:focus {
  background-color: #f9dc4a;
  background-image: linear-gradient(to top, #000 3px, #f9dc4a 3px, #f9dc4a);
  color: #000;
  text-decoration: none;
}
```

If there's something else I should be doing to avoid this issue please let me know! Using the other classes (`heading-anchor`, `with-icon--before`, `with-icon--after`) did not seem appropriate.
